samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>At line 4 (before creating the set), add a comment saying that you are collecting the list of web maps and express maps into a set object. Explain that we use a set operator since sets are unique as we only need to clone those items once.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done!</p>

Reply:
<p>Added additional text to comment.</p>